tools/nl/loadtest/load.py

```diff
@@ -48,12 +48,16 @@ def _run_query(query):
     response = session.get(url, timeout=None)
     if response.status_code == 200:
       debug_timing = response.json().get('debug', {}).get('timing', {})
-      return {
+      result = {
           _RESULT_KEY_CODE: str(response.status_code),
           _RESULT_KEY_NL_TIME: debug_timing.get('getNlResult'),
           _RESULT_KEY_TOTAL_TIME: debug_timing.get('total'),
-          _RESULT_KEY_TILE_TIME: debug_timing.get('getTileResults')
+          _RESULT_KEY_TILE_TIME: debug_timing.get('getTileResults'),
       }
+      result.update(response.json().get('debug',
```

Comment:
why there are two 'debug' key here?

Reply:
that's just the format of the API response